src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -57,27 +58,28 @@
                 ng-repeat="image in ctrl.replicaSetDetail.containerImages">
             {{image}}
           </span>
-          <span class="kd-replicasetdetail-sidebar-title">Service</span>
-          <span class="kd-replicasetdetail-sidebar-line">Label selector</span>
-          <span class="kd-replicasetdetail-sidebar-subline"
-                ng-repeat="service in ctrl.replicaSetDetail.services">
-            {{ctrl.formatLabelString(service.selector)}}
-          </span>
-          <span class="kd-replicasetdetail-sidebar-line">Internal endpoint</span>
-          <div class="kd-replicasetdetail-sidebar-subline"
-               ng-repeat="service in ctrl.replicaSetDetail.services">
-            <span ng-show="service.internalEndpoint">
-              {{service.internalEndpoint}}
+          <div ng-show="ctrl.replicaSetDetail.services" layout="column">
+            <span class="kd-replicasetdetail-sidebar-title">Service</span>
+            <span class="kd-replicasetdetail-sidebar-line">Label selector</span>
+            <span class="kd-replicasetdetail-sidebar-subline"
+                  ng-repeat="service in ctrl.replicaSetDetail.services">
+              {{ctrl.formatLabelString(service.selector)}}
             </span>
-            <i ng-show="service.internalEndpoint"
-               class="material-icons kd-replicasetdetail-sidebar-service-icon">
-              help
-            </i>
-            <span ng-hide="service.internalEndpoint">none</span>
-          </div>
-          <span class="kd-replicasetdetail-sidebar-line">External endpoint</span>
-          <div class="kd-replicasetdetail-sidebar-subline"
-               ng-repeat="service in ctrl.replicaSetDetail.services">
+            <span class="kd-replicasetdetail-sidebar-line">Internal endpoint</span>
+            <div class="kd-replicasetdetail-sidebar-subline"
+                 ng-repeat="service in ctrl.replicaSetDetail.services">
+              <span ng-show="service.internalEndpoint">
```

Comment:
Hmm... You do two ng-shows on the same condition. How about wrapping it with div and having ng-show there?